### PR TITLE
Fix container routes bug

### PR DESCRIPTION
**Pull request by AlfonsoUceda**
Fix a bug with container arch.

There was a problem duplicating the `at` option in routes url.

### CODE REVIEW

File: test/prefix_test.rb [hunk 1]
```diff
@@ -95,7 +95,7 @@ def call(env)
 
     %w(GET POST PUT PATCH DELETE TRACE OPTIONS).each do |verb|
       it "recognizes requests (#{ verb })" do
-        env = Rack::MockRequest.env_for('/admin/home', method: verb)
+        env = Rack::MockRequest.env_for('/home', method: verb)
```
Comment (jodosha) [inline test/prefix_test.rb]: @AlfonsoUceda Why have you removed the prefix from the test?

**Comment by jodosha** [comment]
@AlfonsoUceda This PR doesn't reproduce the problem, and it removes the prefix from the existing tests.

We don't have the proof that it solves the problem.

File: test/prefix_test.rb (hunk 1 above)
Comment (AlfonsoUceda) [inline test/prefix_test.rb]: ok, with the problem, now path_prefix is only for path and url generation, if we use like namespace we have problems with lotus architecture

Then we can use namespace if we want that namespace in the request url and path_prefix only for path and url helpers

**Comment by AlfonsoUceda** [comment]
@jodosha what do you think to merge this? I'll create lotus PR testing this behaviour

File: lib/lotus/router.rb [hunk 2]
```diff
@@ -902,7 +902,7 @@ def call(env)
     #   router.path(:login, return_to: '/dashboard') # => "/login?return_to=%2Fdashboard"
     #   router.path(:framework, name: 'router')      # => "/router"
     def path(route, *args)
-      @router.path(route, *args)
+      @router.custom_path(route, *args)
```
Comment (joneslee85) [inline lib/lotus/router.rb]: @AlfonsoUceda can you please explain why you renamed it to `custom_`?

**Comment by joneslee85** [comment]
@AlfonsoUceda I've hand-tested this and it works nicely

File: lib/lotus/router.rb (hunk 2 above)
Comment (AlfonsoUceda) [inline lib/lotus/router.rb]: it's very weird but `path` didn't call :S and the custom exception didn't raise, that's why I fix exception tests.

**Pull request merged**